src/sentry/static/sentry/app/components/events/interfaces/cspHelp.jsx

```diff
@@ -3,83 +3,68 @@ import React from 'react';
 
 import {t} from 'app/locale';
 
-// TODO: Not sure what we're going to do here re: translations. - Ben
-
 const help = {
   'base-uri': t(
-    `
-The <code>base-uri</code> directive defines the URIs that a user agent
+    `The <code>base-uri</code> directive defines the URIs that a user agent
```

Comment:
newlines can cause issues, not sure how we can make this consistent.

Reply:
We do this a _lot_, we'll have to do something about this

Reply:
Oh I was just looking at this code the other day too, we should convert this stuff to `tct`